Fig4_PyGMT_pandas.py

```diff
@@ -0,0 +1,62 @@
+import io
+
+import pandas as pd
+import pygmt
+import requests
+from pygmt.params import Box
+
+params = {
+    "format": "csv",
+    "starttime": "2000-01-01",
+    "endtime": "2025-10-30",
+    "mindepth": 70,
+    "minmagnitude": 5,
+    "orderby": "magnitude",

```

Comment:
What about removing this parameter from the query and instead sorting it in pandas? 
```suggestion
```
The in the main text, we can add a sentence like:
> We also sort the records in descending order of magnitude so that larger earthquakes (and thus larger circles) are plotted first, preventing them from obscuring smaller ones.